### ISSUE TITLE
Make "Information" text editable after competition posting

### ISSUE THREAD

**Issue by KitClement** [thewca/worldcubeassociation.org#1617]
Currently, the "Information" field on the WCA Event information page is not editable once a competition is announced except for admins/board. Since we have tabs integrated now that can be edited once a competition is announced, I feel it would make sense that this field could be editable as well. If the board has other opinions, that's fine, but I figured this should  be considered.

**Comment by jfly** [comment]
I'm supportive of this change. @KitClement, could you reach out to the Board and get approval for it?

**Comment by KitClement** [comment]
Email sent.

**Comment by KitClement** [comment]
Blocked by #1325.

**Comment by UnsolvedCypher** [comment]
So it sounds like the Board rejected this idea and it can be closed?

**Comment by AlbertoPdRF** [comment]
Hm, yes, I believe this issue can be closed.

**Comment by jfly** [comment]
Sounds good. Closing. @KitClement, feel free to reopen or create a new issue if you think we should be doing anything here.
